qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -746,6 +754,10 @@ def _set_units(self, unit: str) -> None:
         for param in [self.X, self.Y, self.R, self.sensitivity]:
             param.unit = unit
 
+    def _get_complex_voltage(self) -> complex:
+        x, y = self.get_values("X", "Y")
```

Comment:
sr830 does not implement the ``get_values`` method (see it's implementation in sr860 https://github.com/QCoDeS/Qcodes/blob/71823308a3a9235580b09b29077eeccb6419e6a2/qcodes/instrument_drivers/stanford_research/SR86x.py#L1015), so the ``get_values`` equivalent needs to be implemented for sr830 (perhaps it's the same visa command as in sr860? would need to check the manual)

Reply:
@astafan8 good catch, looks like the 830 does have the same `SNAP?` command that the 860 uses. I pulled it over but we should probably test the hardware before merging.